Migration.Executor.WebJob/ItemResponseExtensions.cs

```diff
@@ -10,7 +10,7 @@ namespace Migration.Executor.WebJob
     public static class ItemResponseExtensions
     {
         public static Task<OperationResponse<T>> CaptureOperationResponse<T>(
```

Comment:
Now that you are dealing with ResponseMessage, then the whole wrapping for exceptions won't really do much, right? The failure is based on the ResponseMessage statuscode, the Task (unless some exceptional error) won't throw.